languages/ruby/exercises/concept/conditionals-ternary/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+In this exercise you will rewrite `if/else` statements from a movie theater's website into ternary conditionals.
+
+## 1. Check if a moviegoer is entitled to the seniors' discount
+
+Rewrite the `Moviegoer#ticket_price` method to utilize the ternary operator.
+
+```ruby
+  def ticket_price
+    if age < 60
+      15
+    else
+      10
+    end
+  end
```

Comment:
Rather than the re-phrased answer here, maybe just put an example usage with the output:

```suggestion
new Moviegoer(20, false).ticket_price
# => 15
new Moviegoer(77, false).ticket_price
# => 10
```